truss/remote/baseten/remote.py

```diff
@@ -360,19 +374,39 @@ def _patch(
                 "Failed to calculate patch. Change type might not be supported.",
             )
 
-        if (
-            patch_request.prev_hash == patch_request.next_hash
-            or len(patch_request.patch_ops) == 0
-        ):
+        requires_sync = (
```

Comment:
just so I understand the semantic, `requires_sync` means that there are unapplied patches?

Reply:
could we name it that way?

Reply:
specifically that django has patches that are unapplied to the truss container

Reply:
yeah i think so